code/__HELPERS/atmospherics.dm

```diff
@@ -0,0 +1,51 @@
+/** A simple rudimentary gasmix to information list converter. Can be used for UIs.
```

Comment:
This file needs to be renamed, perhaps to `atmospherics_helpers.dm`. It's failing linters